operations/customers.md

```diff
@@ -160,7 +160,8 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
 | `Title` | string [Title](#title) | optional | Title prefix of the customer. |
 | `Sex` | string [Sex](#sex) | optional | Sex of the customer. |
 | `NationalityCode` | string | optional | ISO 3166-1 code of the [Country](countries.md#country). |
-| `LanguageCode` | string | optional | Language and culture code of the customers preferred language. E.g. `en-US` or `fr-FR`. |
+| `PreferredLanguageCode` | string | optional | Language and culture code of customer's preferred language as set via Operations or API (i.e. without customer's choice). For example: `en-GB`, `fr-CA`. |
```

Comment:
I've got to be honest, I don't understand the explanations here. The word 'preferred' suggests that this is what the customer prefers, but that's the opposite of what it says.